lib/crypto.js

```diff
@@ -17,13 +18,18 @@ const getFipsCrypto = binding.getFipsCrypto;
 const setFipsCrypto = binding.setFipsCrypto;
 
 const Buffer = require('buffer').Buffer;
-const constants = require('constants');
 const stream = require('stream');
 const util = require('util');
 const LazyTransform = require('internal/streams/lazy_transform');
 
 const DH_GENERATOR = 2;
 
+Object.defineProperty(exports, 'constants', {
```

Comment:
The constants themselves have always been read-only.

```js
> const c = require('constants');
undefined
> c.F_OK
0
> c.F_OK = 'a'
'a'
> c.F_OK
0
> 
```

Sure someone could *add* new properties to these objects but the existing values cannot be overwritten.